src/app/components/server/button.module.scss

```
@@ -19,7 +19,7 @@
     }
 
     &:focus {
-      outline: 3px solid $color-blue-30;
+      outline: $boder-focus-width solid $color-blue-30;
```

globlal: Typo on "border" (looks like 7 instances in this PR).

```suggestion
      outline: $border-focus-width solid $color-blue-30;
```

---

Fixed in [d5df068](https://github.com/mozilla/blurts-server/pull/3222/commits/d5df068d91df6ba685c8fef9a895ea4bf53c27d3).